src/cleanvision/imagelab.py

```diff
@@ -277,9 +277,6 @@ def _visualize(
                     filepaths=sorted_filepaths,
                     ncols=self.config["visualize_num_images_per_row"],
                     cell_size=cell_size,
```

Comment:
have you checked the code works if some images are grayscale, eg. running CleanVision on MNIST?

Reply:
Yes, it works. The check is now in this [line](https://github.com/cleanlab/cleanvision/pull/99/files#diff-6c1df1b10a098d41817df84dea378a7e519e90791ecdb2baa738bdbdba027b2eR27)